diff -ru usb-modeswitch-2.5.2~/Makefile usb-modeswitch-2.5.2/Makefile
--- usb-modeswitch-2.5.2~/Makefile	2017-12-31 01:34:02.000000000 -0800
+++ usb-modeswitch-2.5.2/Makefile	2020-04-03 14:41:13.334956116 -0700
@@ -9,7 +9,7 @@
 ETCDIR      = $(DESTDIR)/etc
 SYSDIR      = $(ETCDIR)/systemd/system
 UPSDIR      = $(ETCDIR)/init
-UDEVDIR     = $(DESTDIR)/lib/udev
+UDEVDIR     = $(DESTDIR)/usr/lib/udev
 SBINDIR     = $(PREFIX)/sbin
 MANDIR      = $(PREFIX)/share/man/man1
 VPATH       = jimtcl
